generate.py

```diff
@@ -1,8 +1,10 @@
-import os
+from __future__ import print_function

```

Comment:
What's the purpose of this? I don't see it being used anywhere, but I've seen the `__future__` syntax elsewhere.